fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -1340,6 +1340,10 @@ private PublishResult finishCheckQuorumReplicas(TransactionState transactionStat
                     tabletWriteFailedReplicas.clear();
                     tabletVersionFailedReplicas.clear();
                     for (Replica replica : tablet.getReplicas()) {
+                        if (publishTasks.get(replica.getBackendId()) == null) {
```

Comment:
don't set errorReplicaIds here,  even no this backend's publish task,  it may still ok for this replica.

errorReplicas will update in checkReplicaContinuousVersionSucc